### PR TITLE
Add A10 Networks Ax load balancers to snmp.yml and generator.yml

### PR DESCRIPTION
**Pull request by ninoabbate**
MIBs can be downloaded directly from the device itself, there is no legal way to have world-readable access to the files.

### CODE REVIEW

File: generator/generator.yml [hunk 1]
```diff
@@ -101,3 +101,17 @@ modules:
...
+# 
+  a10_axlb:
+    walk:
+      - sysUpTime # Uptime
+      - interfaces # Interfaces 
+      - axSysCpu # CPU stats
+      - axSysMemory # Memory stats
+      - axSysDisk # Disk stats
+      - axSysHwInfo # Hardware stats (Temperature, Fans Speed)
+      - axNetwork # Network stats
```
Comment (SuperQ) [inline generator/generator.yml]: Please add an end-of-file newline for consistency.

File: snmp.yml [hunk 2]
```diff
@@ -3136,3 +3136,766 @@ servertech_sentry3:
       type: Integer
     - labelname: outletIndex
       type: Integer
+a10_axlb:
+  walk:
+  - 1.3.6.1.2.1.1.3 # Uptime
```
Comment (SuperQ) [inline snmp.yml]: While I find it super useful, please don't include any hand-edited comments or other tweaks in this file.  Annotations should only exist in the `generator.yml`.

File: generator/generator.yml (hunk 1 above)
Comment (ninoabbate) [inline generator/generator.yml]: Done

File: snmp.yml (hunk 2 above)
Comment (ninoabbate) [inline snmp.yml]: Done

**Comment by brian-brazil** [comment]
Can you add information on where this MIB can be freely downloaded?

**Comment by ninoabbate** [comment]
The MIB files are available for download through the load balancer GUI:
1. Hover over System on the menu bar, then select Monitoring.
2. Click on the SNMP tab, then select SNMP MIB Download from the drop-down menu.
3. Select a target location for the MIB archive file, the click Save.

Unfortunately A10 Networks doesn't share them publicly.

**Comment by SuperQ** [comment]
I posted a request to A10 to make their MIBs available.  We'll see if they respond.

**Comment by brian-brazil** [comment]
Lacking MIBs accessible to the general public, we can't have this in this repo I'm afraid.

**Comment by ninoabbate** [comment]
No problem.

**Comment by SuperQ** [comment]
I tried talking to A10 support to make this possible.  They weren't helpful.

**Comment by ninoabbate** [comment]
They've lost here ;-)

**Pull request closed without merging**